src/components/Tank/index.ts

```diff
@@ -0,0 +1,3 @@
+import Tank from './Tank';
+
+export default Tank;
```

Comment:
- [ ] It's not important to have an index file for all folders. You need it when you have several files, and you need to have a single access point for all of them.